src/main/kotlin/org/arend/yaml/ArendYamlSchemaEnabler.kt

```diff
@@ -0,0 +1,9 @@
+package org.arend.yaml
+
+import com.intellij.openapi.vfs.VirtualFile
+import com.jetbrains.jsonSchema.extension.JsonSchemaEnabler
+
+class ArendYamlSchemaEnabler : JsonSchemaEnabler {
+    override fun isEnabledForFile(file: VirtualFile?) =
+            file?.name == "arend.yaml"
```

Comment:
```suggestion
            file?.name == "FileUtils.LIBRARY_CONFIG_FILE"
```

Reply:
`FileUtils.LIBRARY_CONFIG_FILE` should be without quotes and you need to import `FileUtils`.

Reply:
Well, normally this is just a better way to show suggestions (instead of saying "replacing blablabla with blablablabla when blablabla and blablablabla are complicated stuffs). I never actually commit changes by this UI

Reply:
But push commits locally

Reply:
But you can't do these changes locally on your computer and "suggest" it here, right? So, in this example, I should manually add the import command (and replace `"arend.yaml"` with `FileUtils.LIBRARY_CONFIG_FILE`). Doesn't look particularly convenient.

Reply:
> But you can't do these changes locally on your computer and "suggest" it here, right?

Right.

> Doesn't look particularly convenient.

Well, I'd say it's a matter of taste :)